### PR TITLE
Yaml conf

### PR DESCRIPTION
**Pull request by azzra**
This one is for using a more structured config file, in order to segregate each readers between themselves.

### CODE REVIEW

**Comment by azzra** [comment]
@tmartinfr would be great if you can have a look on it when possible. I think we could refacto the config system with each reader managing it's own config. Right now, `files` & `crontabs` are just simple strings but it will become insufficient when dealing with database

**Comment by azzra** [comment]
@tmartinfr up ?

File: src/config.rs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+impl Config {
+    fn from_readers(readers: HashMap<String, Vec<String>>) -> Self {
+        Config { readers: readers }
+    }
+}
+
+pub fn get_config() -> Config {
+    let mut f = File::open("config.yaml").expect("Cannot open config.yaml");
```
Comment (tmartinfr) [inline src/config.rs:19]: I would prefer the `.yml` file extension if you don't mind

Comment (azzra) [reply src/config.rs:19]: `.yaml` seems to be the recommended way: https://yaml.org/faq.html

Is it a no-go for you ?

Comment (tmartinfr) [reply src/config.rs:19]: yes but nobody use this recommendation :stuck_out_tongue:

Comment (azzra) [reply src/config.rs:19]: We should try to stick to the standards in first place then adapt if there is any drawback

**Pull request merged**